.freeipa-pr-ci.yaml

```diff
@@ -58,11 +58,23 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-28/build_url}'
-        test_suite: test_integration/test_external_ca.py::TestExternalCA test_integration/test_external_ca.py::TestSelfExternalSelf test_integration/test_external_ca.py::TestExternalCAInstall
+        test_suite: test_integration/test_external_ca.py::TestExternalCA
         template: *ci-master-f28
-        timeout: 4200
+        timeout: 3600
         topology: *master_1repl_1client
 
+  fedora-28/external_ca_self_install:
```

Comment:
Just call it `external_ca_2` since the self scenario is not the only one there anyway.